src/Control/Monad/Aff.purs

```diff
@@ -110,6 +111,9 @@ instance monadErrorAff ∷ MonadError Error (Aff eff) where
 instance monadEffAff ∷ MonadEff eff (Aff eff) where
   liftEff = _liftEff
 
+instance lazyAff ∷ Lazy (Aff eff a) where
+  defer f = pure unit >>= f
```

Comment:
We can define instance `Lazy (m a)` for any `Monad m`

Reply:
I don't think so. For example, if you used Maybe and tried to construct a fix point it would probably blow the stack.

Reply:
It works for any Free-like Monad though.

Reply:
Where binds are suspended in the data structure, rather.